src/common/webpack/config.ts

```diff
@@ -489,6 +499,14 @@ function getCssLoaders({isEnvDevelopment, isEnvProduction, config}: HelperOption
             esModule: false,
             sourceMap: !config.disableSourceMapGeneration,
             importLoaders: 2,
+            ...(isCssModuleRules
+                ? {
+                      modules: {
+                          localIdentName: '[name]__[local]--[hash:base64:5]',
+                          exportLocalsConvention: 'asIs',
+                      },
+                  }
+                : {}),
```

Comment:
Let's use these options and no other changes will be required.
```suggestion
            modules: {
                auto: true, // uses modules only with files *.module.s?css
                localIdentName: isEnvDevelopment
                    ? '[name]__[local]___[hash:base64:5]'
                    : '[hash:base64]',
                exportLocalsConvention: 'camelCase', // gives both camel case and as is
            },
```

Reply:
Don't you think readable class names are useful for non-dev environments as well?
I see no harm in exposing those in production builds.

Reply:
We minify all other code except class names, which seems strange to me. And here are source-maps to see original classnames.

Reply:
Well, we don't minify BEM classnames. I really doubt we minify css custom property names.

If we were to compare those side-by-side, the difference to total file size seems negligeable:
```css
.MnET1WGLIieyGuzfo5cw {}
.SomeComponent_wrapper-f1da4 {}
```

Hashes generally have more entropy than variable (or file) names, which leads me to suggest total amount of bytes transferred over network (after gzip/brotli) will be even less for readable names.

Reply:
Here's an example why source maps do not help in this case.
How can i attribute those classes to something in my code?
<img width="308" alt="Screen Shot 2024-06-18 at 15 14 43" src="https://github.com/gravity-ui/app-builder/assets/22050667/fba7b456-fcef-4ee8-bebd-dabfc2b6c9ed">